tests/foreman/ui/test_host.py

```diff
@@ -507,6 +509,84 @@ def test_positive_create(self):
             )
             self.assertIsNotNone(search)
 
+    @tier3
+    def test_positive_create_with_inherited_params(self):
+        """Create a new Host in organization and location with parameters
+
+        :BZ: 1287223
+
+        :id: 628122f2-bda9-4aa1-8833-55debbd99072
+
+        :expectedresults: Host has inherited parameters from organization and
+            location
+
+        :CaseImportance: High
+        """
+        org_name = gen_string('alphanumeric')
+        loc_name = gen_string('alphanumeric')
+        org_param_name = gen_string('alphanumeric')
+        loc_param_name = gen_string('alphanumeric')
+        org_param_value = gen_string('alphanumeric')
+        loc_param_value = gen_string('alphanumeric')
+        with Session(self) as session:
+            make_org(
+                session,
+                org_name=org_name,
+                params=[(org_param_name, org_param_value)],
+            )
+            session.nav.go_to_select_org(org_name)
```

Comment:
why not use set_content from factory?

Reply:
Hm... Same question to you - why `set_context` instead of `go_to_select_org`? 🙂 `git grep` shows  `go_to_select_org` is much more preferable way; they both are doing pretty similar things though and `set_context` even has it's own implementation of `force_context` (imo that should be refactored and `set_context` should reuse go_to helpers' force arg). Any real benefits from using it? For me it like 2 helpers doing pretty much the same.

Reply:
Maybe originally go_to helpers had no force_context arg and that's why `set_context` was introduced, but right now when both helpers support forcing context - i can't see much need in `set_context`, it's a duplication.

Reply:
you omitted to say that set_content can also set the location, so we have one function that set org and loc when indicated, I was thinking about the uniformity of the code not more not less.

Reply:
this is not that important, as already many tests are using go_to_select_org.
But we have to make some standard way to set that context and prefer the set context one because it verify if org or loc are already selected before trying to select, and stick well with the concept of context org,loc that we are working in.